src/main/java/ch/njol/skript/lang/ReplacementTypes.java

```diff
@@ -0,0 +1,7 @@
+package ch.njol.skript.lang;
```

Comment:
Does not make sense to put this enum in the ``lang`` package. Please refer to what I told you: ``ch.njol.skript.util``

Reply:
Oop my bad, I put it in the wrong place